lib/models/calendar.ts

```diff
@@ -166,7 +166,9 @@ export class Calendar implements BaseCalendar {
           // Retrieve calendar metadata from the proper of time
           const calendar: RomcalCalendarMetadata = isFromProperOfTime
             ? this.#buildCalendarMetadata(def, date, baseData)
-            : baseData!.calendar;
+            : // TODO: refactor this to avoir non-null assertion
```

Comment:
Not important (as it should be later removed), but still:

```suggestion
            : // TODO: refactor this to avoid non-null assertion
```